styles/style.css

```diff
@@ -6,3 +6,150 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  box-sizing: border-box;
+}
+body {
+  font-family: Circular, spotify-circular, Helvetica, Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  width: 100vw;
```

Comment:
The width here set to 100vw makes your website horizontally scrollable on some web browser. It is thus better to use 100% instead of 100vw, to prevent this from happening.